src/tests/webserver/WebServer_Robustness_Test_001.test.js

```diff
@@ -0,0 +1,47 @@
+import constants from '../../commonMethods/constants'
+import { pluginActivate, pluginDeactivate } from '../../commonMethods/controller'
+import { getCpuLoad } from '../../commonMethods/deviceInfo'
+
+export default {
+  title: 'WebServer Plugin robustness test',
+  description: 'Activates and Deactivates WebServer plugin repeatedly and checks the functionality',
+  context: {
+    cpuLoad: 90,
+  },
+  steps: [
+    {
+      title: 'Activate and deactivate WebServer plugin for 30 times',
+      description: 'Nested test to repeat the test for 30 times',
+      repeat: 30,
+      steps: [
+        {
+          description: 'Deactivate WebServer plugin and check if it is deactivated',
+          test: pluginDeactivate,
+          params: constants.webServerPlugin,
+          assert: 'deactivated',
+        },
+        {
+          description: 'Activate WebServer Plugin and check if it is activated',
+          test: pluginActivate,
+          params: constants.webServerPlugin,
+          assert: 'activated',
+        },
```

Comment:
Same here

Reply:
'pluginActivate' activates the plugin and also **returns the state of the plugin** and it is asserted **using the state as 'activated'**. I think no need of getting the list of active plugins from controller. Please let me know if we can still do as you mentioned.